### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -60,7 +61,7 @@ ram.runtime = "50M"
     main.url = "/"
 
     [resources.apt]
-    packages = "mariadb-server php7.4-fpm"
+    packages = "mariadb-server, php8.2-fpm"
```
Comment (alexAubin) [inline manifest.toml]: zblerg apparently it doesnt like the new php version (from the CI) :

>PHP message: PHP Fatal error:  Array and string offset access syntax with curly braces is no longer supported in /var/www/fluxbb/include/functions.php on line 1673

File: manifest.toml [hunk 2]
```diff
@@ -48,8 +49,8 @@ ram.runtime = "50M"
 
 [resources]
         [resources.sources.main]
-        url = "https://github.com/fluxbb/fluxbb/archive/refs/tags/fluxbb-1.5.11.tar.gz"
-        sha256 = "af464ebebb739a9aec7afbe30edf0c6afdcd7df711a73f98fb814b2bffe4bb9e"
+        url = "https://github.com/fluxbb/fluxbb/archive/refs/tags/fluxbb-1.4.12.tar.gz"
+        sha256 = "c0bb70979f80576f2b078a736cd7b0b7f3dee891006c07f7cfb66ca2ac6d0162"
```
Comment (alexAubin) [inline manifest.toml]: hmm the version a few lines before says 1.5.12 but here it's 1.4.12 ?

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Review by alexAubin — APPROVED** [review]

**Pull request merged**